seers/seer.py

```diff
@@ -0,0 +1,72 @@
+class data_collector_plugin(object):
+	def __init__(self):
```

Comment:
documentation on the object and the rest of the functions/object declarations in this file